io.github.dbchoco.Salawat.yml

```
@@ -0,0 +1,32 @@
+app-id: io.github.dbchoco.Salawat
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: salawat
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: salawat
+    buildsystem: simple
+    build-commands:
+      - install --directory --mode=0755 /app/salawat
+      - install -Dm644 --target-directory=/app/salawat/ Salawat/*
+      - install -Dm755 salawat.sh /app/bin/salawat
+      - install -Dm644 Salawat/icon.png /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
```

```suggestion
      - install -Dm644 Salawat/icon.png /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
```

it was a 512x512 last time I checked.

---

You're right it's 512x512